content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1922,6 +1953,33 @@ model User {
 </tab>
 </TabbedContent>
 
+##### Default value for a scalar list
```

Comment:
Maybe add an emum to the example?

Like

```
enum Role {
  USER
  DEVELOPER
  ADMIN
}
model User {
  id    Int    @id @default(autoincrement())
  roles Role[] @default([USER, DEVELOPER])
}
```

Reply:
Added the enum example